second_problem.rb

```diff
@@ -0,0 +1,32 @@
+class SecondProblem
+
+  attr_reader :never_ever_used_value
+
+  def initialize(numbers_array)
+    @numbers = numbers_array
+  end
+
+  def getBiggestTwoNumbers
+    minValue = -1234567890
```

Comment:
How did you come up with this number?

Reply:
my version of MAXINT since it doesn't have one. didn't want to bloat the solution with creating it